### PR TITLE
feat(fe2): Cancel ongoing measurement with esc

### PR DESCRIPTION
**Pull request by andrewwallacespeckle**
## Description & motivation
If a measurement has started by the user clicking the first point, we know listen for escape. If the user hits escape, it will cancel the ongoing measurement, without disabling Measure mode entirely.

### CODE REVIEW

**Review by benjaminvo — CHANGES_REQUESTED** [review]
You can no longer exit the Measure tool with the esc button. While you're not measuring.

File: packages/frontend-2/lib/viewer/composables/ui.ts
Comment (fabis94) [inline packages/frontend-2/lib/viewer/composables/ui.ts:383]: I don't think this is actually reactive, cause the viewer certainly isn't. Does it really update in real-time? You can validate this by setting up a watcher on this that emits the value in the console.

If it isn't reactive, better to be clear about that and expose this as a function instead (getActiveMeasurement(): boolean)

File: packages/frontend-2/components/viewer/Controls.vue
Comment (fabis94) [inline packages/frontend-2/components/viewer/Controls.vue:394]: Not ideal that the logic for what should happen on Esc is split apart in multiple files (this and postSetup.ts), but it can be left as is for now

**Review by fabis94 — DISMISSED** [review]
I asked 1 question, but outside of that everything looks ok

File: packages/frontend-2/components/viewer/Controls.vue
Comment (andrewwallacespeckle) [reply packages/frontend-2/components/viewer/Controls.vue]: Now in 1 file - Controls.vue

**Review by fabis94 — APPROVED** [review]

**Comment by andrewwallacespeckle** [comment]
@benjaminvo For some reason, I can't merge this until you have approved the changes, even though Fabs already has. Can you re-approve and I'll merge.

**Review by benjaminvo — APPROVED** [review]

**Comment by benjaminvo** [comment]
@andrewwallacespeckle There you go!

**Pull request merged**
